src/java/za/co/mip/ablduck/models/data/SubsectionDataObject.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2017 MIP Holdings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package za.co.mip.ablduck;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') This file "SubsectionDataObject.java" should be located in "za/co/mip/ablduck" directory, not in "/ebs/ebs1/jenkins/workspace/PCTNewJob_PR-247-X33Q424TCA4ZMZR72S4Y6PBXSKT7FL6QYGBFILF26W7CMTBRNBCA/src/java/za/co/mip/ablduck/models/data". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1598)